qemu/tests/cfg/sr_iov.cfg

```diff
@@ -29,35 +29,39 @@
             nics = 'nic1 nic2'
             pci_assignable = vf
             image_snapshot = yes
-        - vf_hotplug:
+        - sriov_hotplug:
             type = sr_iov_hotplug
             nics = nic1
             pci_assignable_nic1 = no
             reference_cmd = lspci
-            find_pci_cmd = 'lspci | tail -n1'
             pci_test_cmd = 'echo %s; nslookup www.redhat.com'
             pci_num = 2
-            repeat_times = 2
             wait_secs_for_hook_up = 3
             assign_static_ip = "yes"
+            variants:
+                - vf_hotplug:
+                    repeat_times = 2
+                - vf_hotplug_500:
+                    repeat_times = 500
+                    test_timeout = 10000
+            variants:
+                - with_hugepage:
+                    only hugepages
+                - without_hugepage:
+                    only smallpages
+            variants:
+                - normal:
```

Comment:
prefer to use  `- @ normal:`

Reply:
@yanan-fu sure, but I am curious to know its significance.

Reply:
The variants "normal" has nothing special operation, with "@" it will not be shown in the case name, i think it is pithy.